src/card.scss

```diff
@@ -364,4 +371,23 @@ $fd-card-default-body-background-color: var(--sapTile_Background) !default;
       padding: 1rem;
     }
   }
+
+  &__loader {
+    @include fd-reset();
+    @include fd-flex-center();
+
+    min-height: 5rem;
+  }
+
+  .#{$numeric-content}__scale-container {
+    @include fd-reset();
+
+    flex: auto;
+  }
+
+  .#{$numeric-content}__kpi {
+    @include fd-reset();
+
+    font-size: 2.25rem;
+  }
```

Comment:
You can't have `numeric-content` classes inside Card component, this is violating the self-contained rules.

Reply:
this class is not needed. now using correct css class in ngx.